MANIFEST.in

```diff
@@ -1,7 +1,7 @@
 include *.go pylib/*.c pylib/*.go
-recursive-include vendor/github.com/docker/docker/builder/dockerfile/command *.go
-recursive-include vendor/github.com/docker/docker/builder/dockerfile/parser *.go
-include vendor/github.com/docker/docker/pkg/system/init_*.go
-include vendor/github.com/docker/docker/pkg/system/lcow*.go
-include vendor/github.com/docker/docker/pkg/system/syscall_*.go
+recursive-include vendor/github.com/moby/buildkit/frontend/dockerfile/command *.go
+recursive-include vendor/github.com/moby/buildkit/frontend/dockerfile/parser *.go
+include vendor/github.com/moby/moby/pkg/system/init_*.go
+include vendor/github.com/moby/moby/pkg/system/lcow*.go
+include vendor/github.com/moby/moby/pkg/system/syscall_*.go
```

Comment:
if I'm reading the updated code correctly, we shouldn't need `moby/moby` any more since they've factored out the specific bits into just the buildkit repo 🎉  -- this probably means you can delete the `moby` submodule and the references here